torchtune/modules/tanh_gate.py

```diff
@@ -22,6 +22,6 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             x (torch.Tensor): input tensor to gate
 
         Returns:
-            torch.Tensor: The output tensor after gating.
+            torch.Tensor: The output tensor after gating. Has the same shape as x
```

Comment:
```suggestion
            torch.Tensor: The output tensor after gating. Has the same shape as ``x``.
```

maybe I'm getting carried away